Implementado os métodos de "Video Resize"

patrickpissurno opened the pull request:
window.set_fullscreen()
window.restoreScreen()
window.set_resolution(width, height)

Propriedade 'window.fullscreen_enabled' foi adicionada para saber se está ou não em tela cheia

patrickpissurno committed:
Implementado os métodos de "Video Resize"

window.set_fullscreen()
window.restoreScreen()
window.set_resolution(width, height)

gcrsaldanha reviewed (changes requested):
Alterar 
    def restoreScreen
para
    def restore_screen

patrickpissurno committed:
Renomeado restoreScreen para restore_screen

O método window.restoreScreen foi renomeado para window.restore_screen para concordar com as convenções do projeto

patrickpissurno commented:
Resolvido
